ddtrace/settings.py

```diff
@@ -0,0 +1,4 @@
+PROCESSING_PIPELINE_KEY = "PROCESSING_PIPELINE"
```

Comment:
we may call that module `constants.py` since it contains (at least for now) only key strings. Also, in general, prefer `'` over `"` for consistency with the majority of modules in this project (at some point I will enforce via a tool one or the other!).